source/common/http/conn_manager_impl.cc

```diff
@@ -1442,6 +1494,16 @@ void ConnectionManagerImpl::ActiveStream::maybeEndEncode(bool end_stream) {
   }
 }
 
+bool ConnectionManagerImpl::ActiveStream::handleDataIfStopAll(ActiveStreamFilterBase& filter,
+                                                              Buffer::Instance& data,
+                                                              bool& filter_streaming) {
+  if (filter.stoppedAll()) {
+    filter.commonBufferDataIfStopAll(data, filter_streaming);
```

Comment:
Can you inline this function here and remove the function? I think it's only called from here? I think will make it a bit easier to read the logic.

Reply:
Fixed. Thanks!